Sprint-1/explore/objects.md

```diff
@@ -5,12 +5,17 @@ In this activity, we'll explore some additional concepts that you'll encounter i
 Open the Chrome devtools Console, type in `console.log` and then hit enter
 
 What output do you get?
+* The output shows that it is a function
 
 Now enter just `console` in the Console, what output do you get back?
+* The output shows that console is an object
 
 Try also entering `typeof console`
+* It returns 'object'
 
 Answer the following questions:
 
 What does `console` store?
+* The console object contains methods (or functions)
 What does the syntax `console.log` or `console.assert` mean? In particular, what does the `.` mean?
+* The . calls upon the different methods available to the console object.
```

Comment:
The description does not seem quite correct. Even if you know what `.` is, I still think you can ask ChatGPT to learn how else we can describe it.